src/util/misc.py

```diff
@@ -60,6 +60,10 @@ def __repr__(self):
         return self.name
 
 
+AUTOMATIC = Sentinel("AUTOMATIC")
+AUTO = AUTOMATIC
```

Comment:
```suggestion
AUTOMATIC = AUTO = Sentinel("AUTOMATIC")
```

Reply:
If we want to promote the usage of `AUTO`, we need to also change the name (for the doc)
```suggestion
AUTO = AUTOMATIC = Sentinel("AUTO")
```